.gitignore

```diff
@@ -7,6 +7,9 @@ __pycache__/
 *.py[cod]
 .mypy_cache/
 
+# This file is allowed specifically because it is a required fixture.
```

Comment:
Please, don't forget to remove this line.

Reply:
Ah yes, thanks for pointing this out! Fixed in 820db0c.